### DISCUSSION TITLE
How to turn off output into console?

### DISCUSSION THREAD

**Discussion by Paul6552** [add2cal/add-to-calendar-button#293] [Q&A] [answered]
Unfortunately I couldn't find any information on how to turn off the output of "add-to-calendar-button".

![image](https://user-images.githubusercontent.com/25934591/211515313-085b0a6a-8256-4d94-b402-f25ab9c93e42.png)

Apart from the fact that it is fundamentally bad to output version numbers (attackers can immediately see which version is in use), I don't want any output.

Not really relevant but my code used: (react)
```
    function addToCalendar(von: Date, bis: Date, meisterschaftsname: string){
        atcb_action({
            name: meisterschaftsname,
            startDate: sprintf( "%04d-%02d-%02d", von.getFullYear(),
                (von.getMonth() + 1), von.getDate() ),
            endDate: sprintf( "%04d-%02d-%02d", bis.getFullYear(),
                (bis.getMonth() + 1), bis.getDate() ),
            options: ['Apple', 'Google', 'iCal', 'Microsoft365', 'Outlook.com', 'Yahoo'],
            timeZone: "Europe/Berlin",
            description: "All information for the championship can be found at myapp.org",
            iCalFileName: meisterschaftsname,
        });
    }
```

**Comment by jekuer** [comment, marked as answer]
There will be a new logic to that in the upcoming v2. Default output will only include a short message (basically copyright information, which seems fair for a free product), but no details about events, etc. This will be only logged, if the debug option is set.
